### ISSUE TITLE
Library isn't importable under Python2

### ISSUE THREAD

**Issue by lurch** [waveform80/colorzero#3]
``` python
Python 2.7.6 (default, Jun 22 2015, 17:58:13) 
[GCC 4.8.2] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import lightwave
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "lightwave/__init__.py", line 37, in <module>
    from .decorator import color_conversion, get_converter
  File "lightwave/decorator.py", line 78
    exec(code, namespace)
SyntaxError: unqualified exec is not allowed in function 'make_color_constructors' it is a nested function
```

BTW I realise you've only just started this project, so tell me to go away if it's too early for me to be reporting issues ;)

**Comment by waveform80** [comment]
Nope, never too early! However I did manage to lose track of this one for a bit. After a recent chat with Ben, I've been persuaded I ought to rename the lib colorzero (for a bit of consistency). I've also decided that, while the decorator stuff is all very academically interesting the actual use-case for it is nonsense; i.e. it's very easy to extend the class with a new colour system ... but who cares? Which regular (most likely educational) user is going to need something more than RGB, RGB565, YUV, HLS, HSV, Luv, Lab, or XYZ?

So, I'm going to push some fairly major changes when I get a sec tomorrow which'll remove the decorator stuff (though I'm keeping the conversion functions split out into their own module as it makes the tests much easier).

Anyway - sorry I left these lying around, and my apologies I'm probably going to clobber the PRs into unmergeable state tomorrow - but your input is much appreciated as ever! Might just get this ready for a release shortly though :)
